stylesheets/stylesheet.css

```diff
@@ -0,0 +1,70 @@
+.box {
+  height: 20em;
+  width: 15em;
+  border: 0.1em solid grey;
+  padding-left: 0.3em;
+}
+
+.container {
+  height: 400px;
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+}
+
+img {
+  max-width: 100%;
+  margin: auto;
+}
+
+h2 {
+  color: green;
+}
+
+.checkout {
+  background-color: green;
+  color: white;
+  font-size: 115%
+}
+
+.nav1 {
+  color: darkblue;
+  background-color: grey;
+  font-size: 150%;
+  text-decoration: underline;
+}
+
+.nav2 {
+  color: white;
+  font-size: 150%;
+  text-decoration: underline;
+}
+
+.first {
+  background-color: black;
+}
+
+.third {
+  background-color: blue;
+}
+
+.logo {
+  height: 15em;
+  width: 15em;
+}
+
+.header-img {
+  height: 25em;
+  width: 55em;
+}
+
+.cart-icon {
+  height: 2em;
+  width: 2em;
+}
+
+.dog-img {
+  height: 8em;
+  width: 8em;

```

Comment:
This doesn't work quite right. Youll want to set the width of the image based on the container width.